### PR TITLE
CT-1301 testCreateTableBasetypes for snowflake

### PR DESCRIPTION
**Pull request by zajca**
Jira: CT-1301
KBC: https://github.com/keboola/connection/pull/4886

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/Snowflake/TableDefinitionOperationsTest.php [hunk 1]
```diff
@@ -826,6 +826,140 @@ public function testCreateTableDefaults(): void
...
+                    ],
+                    'basetype' => 'NUMERIC',
+                    'canBeFiltered' => true,
+                ],
+                [
+                    'name' => 'INTEGER',
+                    'definition' => [
+                        'type' => 'NUMBER',
+                        'nullable' => true,
+                        'length' => '38,0',
```
Comment (romanbracinik) [inline tests/Backend/Snowflake/TableDefinitionOperationsTest.php:924]: testy mi prechadzaju ale jak to ze tu je `38,0` a nie `38,9` ?

**Review by romanbracinik — COMMENTED** [review]
cajk jednu vec nechapem :D

File: tests/Backend/Snowflake/TableDefinitionOperationsTest.php (hunk 1 above)
Comment (zajca) [reply tests/Backend/Snowflake/TableDefinitionOperationsTest.php:924]: To platí jen pro basetype numeric pokud zadáš typ přímo tak tam bude to co tam dá SNFLK

**Review by romanbracinik — APPROVED** [review]
vysvetlene, lgtm

**Pull request merged**